compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -851,6 +877,29 @@ TEST(QuantizedModelVerifierTest, LocalCreateDummyConst)
   EXPECT_NO_THROW(create_dummy_const<Type::FLOAT32>(&g, {32, 32}));
 }
 
+TEST(QuantizedModelVerifierTest, LocalCreateConst)
```

Comment:
`QuantizedModelVerifierTest` Test is used for IR(Op) tests. to distinguish this, how about revise?
- use anoter name, maybe `QuantizedModelVerifierHelperTest`
- use `local_create_const`

Reply:
> use anoter name, maybe QuantizedModelVerifierHelperTest

I like this idea :-D

> use local_create_const

It's using a seperate namespace in order to bound the scope. Does it still need `local_` prefix?

Reply:
> Does it still need local_ prefix?

I don't think so. :)